plugins/module_utils/change_request.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+PAYLOAD_FIELDS_MAPPING = dict(
```

Comment:
Do we need an entry for the `on_hold` parameter here? IIRC, ServiceNow API expects and returns strings, so we need to map those values properly.